.github/workflows/deploy.yml

```diff
@@ -2,24 +2,28 @@ name: Deploy
 on:
   push:
     branches: [ main ]
+defaults:
+  run:
```

Comment:
i think should go into the deploy-app job cos its only used there

Reply:
nice, moved 👍